### DISCUSSION TITLE
NX + Angular MFA + Vite

### DISCUSSION THREAD

**Discussion by synerp** [originjs/vite-plugin-federation#359] [Q&A]
Hi there,

I am currently using NX to manage an Angular Micro Frontend as per the guide here: https://nx.dev/recipes/module-federation/dynamic-module-federation-with-angular

I was wondering if anyone has had any success in using vite-plugin-federation with nx and angular instead of webpack?

-Paul

**Comment by Eddielicious** [comment]
I got the same question. I am using a vite project as my shell and want to use a Angular MF as remote. I am able to create a remoteEntry.js but the shell doesnt show my component
